lib/justice_gov_sk/courts/resource_parser.rb

```diff
@@ -0,0 +1,78 @@
+module JusticeGovSk::Courts
+  class ResourceParser
+    def self.parse(html)
+      document = Nokogiri::HTML(html)
+      detail = document.css('.detail .right .inner > .content')
+      contact = detail.at_css('.contact-address')
+      information_center = detail.at_css('.informacne-centrum')
+      registry = detail.at_css('.podatelna')
+      business_registry = detail.at_css('.orsr')
+      coordinates = document.text.scan(/LatLng\("(\d{2}\.\d+)","(\d{2}.\d+)"\)/)[1]
+
+      HTMLCorrector.correct_contact_table(contact.css('table'))
+
+      attributes = {
+        nazov: detail.at_css('h1').text.strip.presence,
```

Comment:
testy pokryly zatial vacsinu pripadov, custom hack stripu zatial riesit nemusime, radsej by som vsetky data postahoval a sledoval ci to robi problemy pocas analyzy dat.